etna/metrics/functional_metrics.py

```diff
@@ -457,7 +454,98 @@ def max_deviation(y_true: ArrayLike, y_pred: ArrayLike, multioutput: str = "join
         return result
 
 
-rmse = partial(mse_sklearn, squared=False)
+def rmse(y_true: ArrayLike, y_pred: ArrayLike, multioutput: str = "joint") -> ArrayLike:
+    """Root mean squared error with missing values handling.
+
+    .. math::
+        RMSE(y\_true, y\_pred) = \\sqrt\\frac{\\sum_{i=1}^{n}{(y\_true_i - y\_pred_i)^2}}{n}
+
+    The nans are ignored during computation. If all values are nans, the result is NaN.
+
+    Parameters
+    ----------
+    y_true:
+        array-like of shape (n_samples,) or (n_samples, n_outputs)
+
+        Ground truth (correct) target values.
+
+    y_pred:
+        array-like of shape (n_samples,) or (n_samples, n_outputs)
+
+        Estimated target values.
+
+    multioutput:
+        Defines aggregating of multiple output values
+        (see :py:class:`~etna.metrics.functional_metrics.FunctionalMetricMultioutput`).
+
+    Returns
+    -------
+    :
+        A non-negative floating point value (the best value is 0.0), or an array of floating point values,
+        one for each individual target.
+    """
+    mse_result = np.asarray(mse(y_true=y_true, y_pred=y_pred, multioutput=multioutput))
```

Comment:
Why do you use `np.asarray` here?

Reply:
To avoid errors from mypy.

Reply:
I'm not sure if it is ok. In `multioutput="joint"` the result should be float, but `np.asarray` makes it `smth like `np.array([value]`)` instead of `value`.
I thought that we have tests that check that.

Reply:
Yes, but `np.sqrt` makes from `np.array(value)` to `np.float`. 
<img width="857" alt="image" src="https://github.com/user-attachments/assets/157aeb25-71e4-4e8a-8b84-8fad1a55cac4" />

Reply:
- We tested that current behavior is ok.
- It could be not so obvious for other readers why do we use `np.asarray`.
- Where exactly mypy is raising error? In some other functional metrics we just put `type: ignore` at the last line to ignore the mypy.